frontend/src/metabase/qb/components/drill/UnderlyingRecordsDrill.jsx

```diff
@@ -27,7 +27,7 @@ export default ({ question, clicked }: ClickActionProps): ClickAction[] => {
         {
             name: "underlying-records",
             section: "records",
-            title: "View " +
+            title: t`View` + " " +
```

Comment:
For this one I think the following should work. I suppose there's a chance for something like this that the broken up words could create some slightly unnatural sentences in certain languages if the proper context / tense isn't presented to the translator but I'm not sure of a way around this.

```jsx
title: t`View ${inflect(t`these`, count, t`this`, t`these`)} ${inflect(query.table().display_name, count}`
```